src/types.mligo

```diff
@@ -122,14 +122,15 @@ type proposal =
 type seconds = nat
 type voting_period = { length : seconds }
 
+// For efficiency, we only keep a `nat` for the numerator, whereas the
+// denominator is not stored and has a fixed value of `1000000`.
+type fraction = { numerator : int }
+let fraction_denominator = 1000000
+let percent_to_fraction = 10000
```

Comment:
I think in the update_quorum procedure, we use this to represent percentages and some intermediate fractional values. It felt odd to use a type with 'quorum_` in it.

Reply:
> we use this to represent percentages and some intermediate fractional values

Yeah, but still only fractions for the same denominator :thinking: 

My point is that this cannot represent any fraction or rational number, but only the ones where the denominator is fixed at 1 million, which is a choice we made for `quorum`s for a practical reason.

This is why I still think it's ok to keep this as `quorum_fraction` (as "a fraction for quorum") and maintain the synonym of `quorum_threshold`.

Reply:
>My point is that this cannot represent any fraction or rational number

AFAIU its a fixed precision representation, so it can represent any fraction to the precision decided by the deniminator. 

But I am fine with changing the name :+1: